openpdf/src/main/java/com/lowagie/text/Document.java

```diff
@@ -101,17 +101,14 @@
     
     // membervariables
     /**
-     * This constant may only be changed by Paulo Soares and/or Bruno Lowagie.
      * @since	2.1.6
      */
```

Comment:
Sure.  Unless you don't like the name :)  I chose open PDF because I wanted this project to remain open and free to all.